packs/data/abomination-vaults-bestiary.db/morlock-engineer.json

```diff
@@ -713,7 +713,7 @@
                     "value": 2
                 },
                 "description": {
-                    "value": "<p>The morlock engineer tinkers with an adjacent construct or mechanical hazard. They attempt a Crafting check against the construct's or hazard's Fortitude DC. The morlock can't get an outcome better than failure if the target's level is more than double the morlock's.</p>\n<p>This ability reflects hasty battlefield repairs; once a construct or hazard regains Hit Points from this ability, it can't do so again until it's been @UUID[Compendium.pf2e.actionspf2e.Repair]{Repaired}.</p>\n<hr />\n<p><strong>Critical Success</strong> The target regains [[/r {8d6}[healing]]]{8d6 Hit Points} and a +1 circumstance bonus to attack rolls for 1 minute. Alternately, the morlock can deal [[/r {8d6}]]{8d6 damage} (bludgeoning, piercing, or slashing, as chosen by the morlock engineer) to the construct or hazard.</p>\n<p><strong>Success</strong> As critical success, but the target regains [[/r {4d6}[healing]]]{4d6 Hit Points} or the morlock deals [[/r {4d6}]]{4d6 damage}.</p>\n<p><strong>Critical Failure</strong> The morlock injures themself, taking [[/r {3d6}]]{3d6 damage} (typically bludgeoning, piercing, or slashing, but potentially a different type at the GM's discretion).</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Effect: Uncanny Tinker]{Effect: Uncanny Tinker}</p>"
+                    "value": "<p>The morlock engineer tinkers with an adjacent construct or mechanical hazard. They attempt a Crafting check against the construct's or hazard's Fortitude DC. The morlock can't get an outcome better than failure if the target's level is more than double the morlock's.</p>\n<p>This ability reflects hasty battlefield repairs; once a construct or hazard regains Hit Points from this ability, it can't do so again until it's been @UUID[Compendium.pf2e.actionspf2e.Repair]{Repaired}.</p>\n<hr />\n<p><strong>Critical Success</strong> The target regains [[/r 8d6[healing]]]{8d6 Hit Points} and a +1 circumstance bonus to attack rolls for 1 minute. Alternately, the morlock can deal [[/r {8d6}]]{8d6 damage} (bludgeoning, piercing, or slashing, as chosen by the morlock engineer) to the construct or hazard.</p>\n<p><strong>Success</strong> As critical success, but the target regains [[/r 4d6[healing]]]{4d6 Hit Points} or the morlock deals [[/r {4d6}]]{4d6 damage}.</p>\n<p><strong>Critical Failure</strong> The morlock injures themself, taking [[/r {3d6}]]{3d6 damage} (typically bludgeoning, piercing, or slashing, but potentially a different type at the GM's discretion).</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Effect: Uncanny Tinker]{Effect: Uncanny Tinker}</p>"
```

Comment:
> Alternately, the morlock can deal [[/r {8d6}]]{8d6 damage} (bludgeoning, piercing, or slashing, as chosen by the morlock engineer)

What about this part? Should it stay as is?

Reply:
Yeah that's fine.

Reply:
At most I think we should just change each damage type to be their own button, but that can be a follow-up.